flake.nix

```diff
@@ -73,6 +73,15 @@
         }).outPath;
       };
 
+      mkDocs = pkgs: with pkgs; writeShellApplication {
+        name = ",docs";
+        runtimeInputs = [
+          pkgs.python3
```

Comment:
You don't _know_ that what you have here is everything needed (although I'd assume it does). Python is also a rather large dependency to pull in.

Reply:
Fair enough, I checked it with `nix-shell -p python3` and it seems to work.